src/Inertia.php

```diff
@@ -13,7 +13,7 @@ public static function render ($template = 'app', $props = [], $viewData = []) {
         ? $template->name() 
         : $template,
       'props' => $props,
-      'url' => page()->inertiaUri()
+      'url' => $request -> url() -> toString()
```

Comment:
Could you remove spaces: `$request->url()->toString()`

Reply:
yep!